.changeset/fifty-colts-cross.md

```diff
@@ -0,0 +1,8 @@
+---
+"@rhds/elements": minor
+---
+
+Renamed `rh-secondary-nav` to `rh-navigation-secondary` 
+
+ - Renamed all sub components `rh-secondary-nav-*` to `rh-navigation-secondary-*` 
+ - Renamed all css custom properties `--rh-secondary-nav-*` to `--rh-navigation-secondary-*`
```

Comment:
```suggestion
---
"@rhds/elements": major
---

Renamed `rh-secondary-nav` to `rh-navigation-secondary` 

 - Renamed all sub components `rh-secondary-nav-*` to `rh-navigation-secondary-*` 
 - Renamed all css custom properties `--rh-secondary-nav-*` to `--rh-navigation-secondary-*`
```
Since we don't alias to the old names this is a breaking change. is secondary nav in use in the wild? we'll have to put out comms. Probably best to provide aliases (in the lightdom stylesheets as well) if `<rh-secondary-nav>` is in use anywhere

Reply:
It is, we'll definitely need to communicate the change.  Will add aliases in lightdom.

Reply:
ok let's also add deprecated tag names to the ts sources like we did for universal footer

Reply:
> ok let's also add deprecated tag names to the ts sources like we did for universal footer

Also, just a reminder, that hasn't actually been done yet for footer.  You might be referring to your comment on Jan 10: https://github.com/orgs/RedHat-UX/discussions/695#discussioncomment-4641724. 

I'll implement this same idea here.

Reply:
yup! that's the one!

Reply:
Ok added that in as well as some CSS prop fallbacks so we aren't completely nuking possible customization there as well.  Also added a deprecated demo with the old tag code to test changes and uncovered a bug that appeared when old tags were used so fixed that as well.